static/app/components/profiling/flamegraphZoomView.tsx

```diff
@@ -336,15 +336,15 @@ function FlamegraphZoomView({
   const selectedFramesRef = useRef<FlamegraphFrame[] | null>(null);
 
   useEffect(() => {
-    if (flamegraphState.profiles.highlightFrame) {
+    if (flamegraphState.profiles.highlightAllFrames) {
       selectedFramesRef.current = flamegraph.findAllMatchingFrames(
-        flamegraphState.profiles.highlightFrame.name,
-        flamegraphState.profiles.highlightFrame.package
+        flamegraphState.profiles.highlightAllFrames.name,
+        flamegraphState.profiles.highlightAllFrames.package
       );
     } else {
       selectedFramesRef.current = null;
     }
-  }, [flamegraph, flamegraphState.profiles.highlightFrame]);
+  }, [flamegraph, flamegraphState.profiles.highlightAllFrames]);
```

Comment:
`highlightAllFrames` sounds like a boolean flag to me - I would prefer to keep this to highlightFrame and change how we read/init qs. For example, if we have highlightFrame[], then you can select one or many frames (and they dont necessarily need to be the same function frames)

Reply:
until we get a solid `frameId` it will always be a highlight all; if we support `highlightFrame[]` then it's going to be a highlight all of many different frames. that's arguably more confusing as we'd probably want to visually differentiate frame `foo` from `bar`. if you do `highlightFrame[]` and we actually want to `highlightAll` of a certain frame, surely we wouldn't have an array of all the instances either, we'd have another piece of state that tells us to highlight all frames matching `xyz`... at that point i'd begin questioning if i want to be using a query-string, i would bet it would be super unwieldy. 

the rename of `highlightFrame` to `highlightAllFrames` is to better represent what our code is actually doing; it's highlighting all frames that match `frameName` and `packageName` and right now it only serves that purpose.

i can rename it back and we can revisit it later when we can better identify individual frames